src/covid_shared/paths.py

```diff
@@ -27,56 +25,77 @@
+# All paths should point here.
+COVID_19 = Path('/ihme/covid-19')
+
+# Shared config for running rclone on the IHME OneDrive
+RCLONE_CONFIG_PATH = COVID_19 / '.config' / 'rclone' / 'rclone.conf'
+
+# Top level directories.  These represent, mostly, outputs of pipeline stages.
+# Keep them in alphabetical order.
+# TODO: This should definitely be in a container.
+AGE_SPECIFIC_RATES_ROOT = COVID_19 / 'age-specific-rates'
+DURATIONS_ROOT = COVID_19 / 'durations'
+EXCESS_MORTALITY_ROOT = COVID_19 / 'excess-mortality'
+GBD_ROOT = COVID_19 / 'gbd'
+HISTORICAL_MODEL_ROOT = COVID_19 / 'historical-model'
+MANDATES_ROOT = COVID_19 / 'mandates'
+MASK_USE_OUTPUT_ROOT = COVID_19 / 'mask-use-outputs'
+MOBILITY_COVARIATES_OUTPUT_ROOT = COVID_19 / 'mobility-covariate'
 MOBILITY_COVARIATES_GPR_OUTPUT_ROOT = MOBILITY_COVARIATES_OUTPUT_ROOT / 'gpr_outputs'
-TESTING_OUTPUT_ROOT = Path('/ihme/covid-19/testing-outputs')
-VACCINE_COVERAGE_OUTPUT_ROOT = Path('/ihme/covid-19/vaccine-coverage')
-SEIR_COVARIATES_OUTPUT_ROOT = Path('/ihme/covid-19/seir-covariates')
-
-SEIR_REGRESSION_OUTPUTS = Path('/ihme/covid-19/seir-regression')
-SEIR_FORECAST_OUTPUTS = Path('/ihme/covid-19/seir-forecast')
-SEIR_FINAL_OUTPUTS = Path('/ihme/covid-19/seir-outputs')
-SEIR_DIAGNOSTICS_OUTPUTS = Path('/ihme/covid-19/seir-diagnostics')
+MODEL_INPUTS_ROOT = COVID_19 / 'model-inputs'
+MORTALITY_AGE_PATTERN_ROOT = COVID_19 / 'mortality-age-pattern'
+MORTALITY_RATIO_ROOT = COVID_19 / 'mortality-ratio'
+PAST_INFECTIONS_ROOT = COVID_19 / 'past-infections'
+PNEUMONIA_OUTPUT_ROOT = COVID_19 / 'pneumonia'
+POPULATION_DENSITY_OUTPUT_ROOT = COVID_19 / 'population-density'
+RESULTRON_ROOT = COVID_19 / 'resultron'
+SEIR_COVARIATE_PRIORS_ROOT = COVID_19 / 'seir-covariate-priors'
+SEIR_COVARIATES_OUTPUT_ROOT = COVID_19 / 'seir-covariates'
+SEIR_DIAGNOSTICS_OUTPUTS = COVID_19 / 'seir-diagnostics'
+SEIR_FORECAST_OUTPUTS = COVID_19 / 'seir-forecast'
+SEIR_FINAL_OUTPUTS = COVID_19 / 'seir-outputs'
+SEIR_REGRESSION_OUTPUTS = COVID_19 / 'seir-regression'
+SHAPEFILE_ROOT = COVID_19 / 'shapefiles'
+SNAPSHOT_ROOT = COVID_19 / 'snapshot-data'
+STATIC_DATA_INPUTS_ROOT = COVID_19 / 'static-data'
+SYMPTOM_SURVEY_ROOT = COVID_19 / 'symptom-survey-data'
+TESTING_OUTPUT_ROOT = COVID_19 / 'testing-outputs'
+UNVERSIONED_INPUTS_ROOT = COVID_19 / 'unversioned-inputs'
+VACCINE_COVERAGE_OUTPUT_ROOT = COVID_19 / 'vaccine-coverage'
+VARIANT_OUTPUT_ROOT = COVID_19 / 'variant-scaleup'
+VISIT_VOLUME_ROOT = COVID_19 / 'visit-volume/'
+WEBSCRAPER_ROOT = COVID_19 / 'webscrape'
+
+######################################
+# Shared file and subdirectory names #
+######################################
+METADATA_FILE_NAME = Path('metadata.yaml')
 
+LOG_DIR = Path("logs")
+LOG_FILE_NAME = Path("master_log.txt")
+DETAILED_LOG_FILE_NAME = Path("master_log.json")
 
-# Shared file names
-METADATA_FILE_NAME = Path('metadata.yaml')
 BEST_LINK = Path('best')
 LATEST_LINK = Path('latest')
 PRODUCTION_RUN = Path('production-runs')
 
+# ???
```

Comment:
Many - but not all - of these are specifically `/model-inputs/` subdirectories. Let's discuss if this is not the appropriate place to organize the ETL outputs.

Reply:
Probably belongs in etl